shared/cfg/guest-hw.cfg

```diff
@@ -181,13 +180,9 @@ variants:
         # Mellanox CX4 MT27700 Family
         # For PowerPC driver (kernel module) that supports SR-IOV hardware is
         # mlx5_core
-        driver = igb
+        #driver = igb
```

Comment:
we don't have to comment the driver's default value, configure the required one and run it.